amass/config.go

```diff
@@ -186,3 +218,108 @@ func (c *Config) GetAPIKey(source string) *APIKey {
 	}
 	return nil
 }
+
+// LoadSettings parses settings from an .ini file and assigns them to the Config.
+func (c *Config) LoadSettings(path string) error {
+	cfg, err := ini.LoadSources(ini.LoadOptions{
+		Insensitive:  true,
+		AllowShadows: true,
+	}, path)
+	if err != nil {
+		return err
```

Comment:
This isn't a blocker for this PR but I just wanted to point that out. I found with Go that errors can sometimes be ambiguous as you don't really know where they were returned. At work, I've opted for doing `return errors.Wrap(err, "reason why I am returning this error")` which shows up as `reason why I am returning this error: whatever the error was`. The errors package can be found at https://github.com/pkg/errors

Reply:
I agree. I usually do wrap error messages, but rushed last night. Thank you for catching it!